cluster-agent/Makefile

```diff
@@ -109,7 +113,7 @@ test: fmt vet envtest ## Run tests.
 ##@ Build
 
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/manager main.go
```

Comment:
Great addition for the CGO_ENABLED=0. Although we don't use any C-Go stuff, this results in smaller size binary and less prone to attack surface, since it eliminates the need to link to C code with know vulnerabilities! :D